src/java/fr/paris/lutece/plugins/forms/web/admin/FormControlJspBean.java

```diff
@@ -812,4 +822,14 @@ private String getControlReturnUrl( HttpServletRequest request )
 
         return url.getUrl( );
     }
+
+    public static String formatDate(String inputDate) {
+
+        DateTimeFormatter inputFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
+        LocalDate date = LocalDate.parse(inputDate, inputFormatter);
+        DateTimeFormatter outputFormatter = DateTimeFormatter.ofPattern("dd/MM/yyyy");
```

Comment:
the model should not contain a formatted date, but a date object,
the date will be formatted by Freemarker later (or maybe in the browser for  FrontOffice unauthenticated features), depending on the locale the user or of the browser.